variables.tf

```diff
@@ -21,18 +21,47 @@ variable "vpc_cidr_block" {
   default     = "10.0.0.0/18"
 }
 
+variable "vpc_id" {
```

Comment:
Do you think we can get fully rid of `create_vpc` in order to simplify the module configuration?
We could assume that if the `vpc_id` is not null, then let's not create a vpc?

Reply:
@eliecharra I think I need to leave it in actually, because of this:

<img width="856" alt="image" src="https://github.com/user-attachments/assets/f7f1d8e3-23be-4c57-b310-b4a44ecf4459" />

So basically if I try to use `var.vpc_id != null` Terraform doesn't necessarily know in advance what the value is going to be in order to generate the plan 😞

Reply:
Ha yeah I remember know, I tried to do it for GCP but hit the same issue. Let's keep it this way, we also do that for GCP so 🤷🏻 

https://github.com/spacelift-io/terraform-google-spacelift-selfhosted/blob/main/modules/network/variables.tf#L5